package.json

```diff
@@ -18,6 +18,7 @@
     "@angular/platform-browser": "^14.1.0",
     "@angular/platform-browser-dynamic": "^14.1.0",
     "@angular/router": "^14.1.0",
+    "bootstrap": "^5.2.2",
```

Comment:
i thought we were going to use material?

Reply:
I followed maximillian's example for project init. Didn't think about material

Reply:
Also do i uninstall bootstrap

Reply:
it will be difficult to have two css frameworks to work with and either ignore one of them or get them conflicting, don't you think?